src/main/java/org/ladocuploader/app/utils/HouseholdUtilities.java

```diff
@@ -53,9 +53,9 @@ public static ArrayList<LinkedHashMap> formattedHouseholdData(Submission submiss
 
   }
 
-  protected static LinkedHashMap householdData(ArrayList<LinkedHashMap> household, String uuid) {
-    LinkedHashMap user = new LinkedHashMap();
-    for (LinkedHashMap hhmember : household) {
+  private static Map<String, Object> householdData(List<Map<String, Object>> household, String uuid) {
+    Map<String, Object> user = new LinkedHashMap<>();
+    for (Map<String, Object> hhmember : household) {
```

Comment:
I don't generally change code outside of the PR, but I had added a method to this class and updated a few other things in the process.  I later took the method out as I didn't really need it. 

I can reverse these if you want, but I think it's good practice to use generic classes versus being so specific.

Reply:
This is great!